xml/System.IO.Compression/BrotliStream.xml

```diff
@@ -79,10 +79,10 @@
         <Parameter Name="leaveOpen" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="stream">To be added.</param>
-        <param name="compressionLevel">To be added.</param>
-        <param name="leaveOpen">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="stream">The stream to compress.</param>
+        <param name="compressionLevel">One of the enumeration values that indicates whether to emphasize speed or compression efficiency when compressing the stream.</param>
+        <param name="leaveOpen"><see langword="true" /> to leave the stream open after disposing the <see cref="T:System.IO.Compression.Brotli.BrotliStream" /> object; otherwise, <see langword="false" />.</param>
```

Comment:
```suggestion
        <param name="leaveOpen"><see langword="true" /> to leave the stream open after disposing the <see cref="T:System.IO.Compression.BrotliStream" /> object; otherwise, <see langword="false" />.</param>
```